### PR TITLE
chore(core): bump package to 0.0.149

### PR DESCRIPTION
**Pull request by anotherchrisberry**
* fix(core): add authorized field to test accounts
* fix(core/accounts): fix versioned account lookup
* fix(core): avoid scrollbars in cluster pod header
* feat(core/pipelines): allow triggering of execution via deep link
* fix(core/stages): Hide excluded providers
* refactor(core/filterModel): Refactor filter.model.service to Typescript
* refactor(core): convert security group filter service to TS
* fix(core): allow overflow on details panel for menus

### CODE REVIEW

**Comment by christopherthielen** [comment]
LGTM nice work

File: app/scripts/modules/core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spinnaker/core",
-  "version": "0.0.148",
```
Comment (christopherthielen) [inline app/scripts/modules/core/package.json]: worst part of old version

File: app/scripts/modules/core/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "0.0.149",
```
Comment (christopherthielen) [inline app/scripts/modules/core/package.json]: best part of pr

**Comment by christopherthielen** [comment]
🍪

File: app/scripts/modules/core/package.json (hunk 2 above)
Comment (anotherchrisberry) [reply app/scripts/modules/core/package.json]: I went back and forth on it before settling on this

**Comment by christopherthielen** [comment]
I sure hope CI passes

**Pull request merged**